Automattic/newspack-ios
# Share Extension Part Four

Refs #91 

This is part four of a four part series of PRs implementing the Photo support via a share extension.

In this PR the final step of the process, shared item reconciliation, is handled in the main app.  Sharing from Photos to Newspack should now work end to end.  
Additionally a number of unit tests are added to test updates to the FolderManager, and the behavior of the shadow model.

To test: 
Run tests and confirm tests pass. 
Share some photos from the photos app to Newspack.  
Launch Newspack and confirm that the photos you shared are present in the target story folder.
Bonus: Check the files app and confirm the photos are there as well.

@jleandroperez would you please?  This is the last one of this batch!
